app/view/pages/github/index.js

```diff
@@ -0,0 +1,95 @@
+import React, {useEffect, useState} from 'react';
+import {Animated} from 'react-native';
+import {useSelector, useDispatch} from 'react-redux';
+import {useNavigation} from 'react-navigation-hooks';
+
+import Item from './Item';
+import {View, FlatList, Loading, Appbar, Button, Text} from 'view/components';
+
+import i18n from 'i18n';
+import style from 'view/style';
+
+import {getRepos} from 'store/reducers/github';
+import {selectGithub} from 'store/selectors/github';
+
+import buttons from './buttons';
+
+const Github = () => {
+  const {appLang, githubLang} = i18n;
+  const {appStyle, githubStyle} = style;
+
+  const [query, setQuery] = useState('react');
+  const [scaleValue] = useState(new Animated.Value(0));
+
+  const dispatch = useDispatch();
+  const {goBack} = useNavigation();
+  const github = useSelector(selectGithub);
+
+  const _goBack = () => goBack();
+
+  useEffect(() => {
+    dispatch(getRepos(query));
+  }, [dispatch, query]);
+
+  useEffect(() => {
+    if (!github.loading && github.data.length) {
+      Animated.timing(scaleValue, {
+        delay: github.data[query] * 350,
+        duration: 600,
+        toValue: 1,
+      }).start();
+    }
+  }, [github, query, scaleValue]);
+
+  return (
+    <View style={appStyle.container}>
+      <Appbar.Header style={appStyle.header}>
+        <Appbar.Action onPress={() => _goBack()} icon="arrow-back" />
+        <Appbar.Content title={githubLang.title} />
+      </Appbar.Header>
+
+      <View style={appStyle.content}>
+        <View style={githubStyle.buttonArea}>
+          {buttons.map(button => (
+            <Button
+              key={button.id}
+              mode="outlined"
+              color={'#13a77f'}
```

Comment:
I'm very surprised. How your linter doesn't show an error to you. line 57 should write like below:

    color="#13a77f"

Reply:
done